data/src/main/java/dev/baseio/discordjetpackcompose/utils/Constants.kt

```diff
@@ -0,0 +1,5 @@
+package dev.baseio.discordjetpackcompose.utils
+
+object Constants {
```

Comment:
@shubhamsinghshubham777 move this constant to the repository impl where we will have the logic to fetch the countries.